astropy/io/fits/fitstime.py

```diff
@@ -347,7 +347,7 @@ def _convert_global_time(table, global_info):
         Global time reference frame information.
     """
     # Read in Global Informational keywords as Time
-    for key, value in global_info.items():
+    for key in global_info.keys():
```

Comment:
Does this not work?
```suggestion
    for key in global_info:
```

Reply:
It certainly might. I'll give it a go. Ruff didn't flag it for change, perhaps because iteration isn't guaranteed to be on the key?

Reply:
✅